utils.go

```diff
@@ -55,7 +55,7 @@ func fromTOML(file string, s *state) (bool, string) {
 	}
 	addDefaultHelmRepos(s)
 	resolvePaths(file, s)
-	if !noEnvSubst {
+	if !noEnvSubst || !noEnvValuesSubst {
```

Comment:
I think this should be `&&` .. the current condition means that if `noEnvSubst = false` and `noEnvValuesSubst = true` , environment variables will still be substituted in values files.

Reply:
Yep, you are right - thanks.